osu.Game/Screens/Select/FooterButtonRandom.cs

```diff
@@ -37,26 +40,11 @@ private void load(OsuColour colours)
             SelectedColour = colours.Green;
             DeselectedColour = SelectedColour.Opacity(0.5f);
             Text = @"random";
-            Hotkey = Key.F2;
-        }
-
-        protected override bool OnKeyDown(KeyDownEvent e)
-        {
-            secondaryActive = e.ShiftPressed;
-            updateText();
-            return base.OnKeyDown(e);
-        }
-
-        protected override void OnKeyUp(KeyUpEvent e)
-        {
-            secondaryActive = e.ShiftPressed;
-            updateText();
-            base.OnKeyUp(e);
         }
 
         private void updateText()
```

Comment:
This change regresses the behaviour of `updateText`. Before, if you only pressed <kbd>Shift</kbd> on song select, the text of the *random* button would change to *rewind*.

With this change, it rapidly changes for a second when the button is pressed, so if you just press it normally, it changes from *random* to *rewind* and then back to *random* almost immediately which looks bad.

Reply:
How would you suggest this gets fixed? I don't think making a keybinding for footer secondary action is very descriptive. Should we just get rid of updateText?

Reply:
Either that, or remove the key binding and always make it shift+random. Not sure which is better to be honest.

Reply:
I feel like forcing shift to be the secondary modifier would almost defeat the purpose of making these rebindable. I'm swinging more towards just removing the text for the time being.

Reply:
Sure, let's go with that for now.